templates/silicon/index.html

```diff
@@ -0,0 +1,550 @@
+{% extends "silicon/base_silicon.html" %}
+
+{% block title %}Silicon Programme{% endblock %}
+
+{% block meta_description %}
+  Unlock new opportunities with optimised secure open source solutions
+{% endblock meta_description %}
+
+{% block meta_copydoc %}https://docs.google.com/document/d/1nyiz4I1GqBiXe_SKpDzho3yv20LKyYpyhHh0fvvUrOI/edit{% endblock meta_copydoc %}
+
+{% block content %}
+
+<section class="p-strip--suru-shape-dark is-deep">
+  <div class="row u-equal-height u-vertically-center">
+    <div class="col-8">
+      <h1 class="p-heading--2 u-no-margin--bottom" style="font-weight: 500;">Silicon programme</h1>
+      <p class="p-heading--2" style="font-weight: 100;">
```

Comment:
If this style of heading is/is going to be repeated we could make a class until it has been fixed upstream in vanilla